### DISCUSSION TITLE
I meet some questions with DistinctTest

### DISCUSSION THREAD

**Discussion by sandexp** [cmu-db/bustub#297] [Q&A]
From the test of project and autograde, I find duplicated distinct test is failed.

But when I use local test(executor_test). 
The assertion failed and show:

Expected equality of these values:
  result_set.size()
    Which is: 100
  10

But I print data to screen, and see data is not duplicated from 1 to 99.
like :

| Col C  = 1 |
| Col C  = 2 |
| Col C  = 3 |
| Col C  = 4 |
......
| Col C  = 97 |
| Col C  = 98 |
| Col C  = 99 |

So I wonder if this unit test is not OK . Hope your sendback, thank you.

**Comment by OneMore14** [comment]
in TableGenerator::GenerateTestTables(), colC in table "test_7" is indeed between 0 and 9.

**Comment by zjy-dev** [comment]
I got the same problem today, The output is excatly from 1 to 99.
So are we wrong about what we think, or indeed it's an issue?
Hope your back message, thank you sir, the course is very helpful.

**Comment by OneMore14** [comment]
> I got the same problem today, The output is excatly from 1 to 99. So are we wrong about what we think, or indeed it's an issue? Hope your back message, thank you sir, the course is very helpful.

your SeqScanExecutor  works not correctly,  you should think about how to utilize the output schema of the SeqScanPlan to get  
the correct output. I was also confused by this problem before, now the DistinctTest has been passed

**Comment by zjy-dev** [comment]
Thank you soooo much bro, it may save me a lot of time, I'm now full scored. 
And for all guys have this problem, check your SeqScan, make sure you've considered the Expressions on the output columns.

**Comment by Rity01** [comment]
I got the the same problem,and I found it is because I use the output schema to cut the tuple in SeqScan,actually I should use the table schema.
